content/chapters/template-chapter/lab/README.md

```diff
@@ -1,58 +1,29 @@
-# Compute
+# Template chapter
 
-The main criterion we use to rank CPUs is their _computation power_, i.e. their ability to crunch numbers and do math.
-Numerous benchmarks exist out there and they are publicly displayed on websites such as [CPUBenchmark](https://www.cpubenchmark.net/).
+This is an instance of a lab placeholder. This will be used to showcase the different highlights of the syntax.
 
-This benchmark measures the performance of the computer's CPU in a variety of scenarios:
-- its ability to perform integer operations
-- its speed in floating point arithmetic
-- data encryption and compression
-- sorting algorithms and others
+In order to use links to to outside resources, you can add them as [such](http://example.com).
 
-You can take a look at what exactly is measured using [this link](https://www.cpubenchmark.net/cpu.php?cpu=AMD+Ryzen+Threadripper+PRO+5995WX).
-It displays the scores obrtained by a high-end CPU.
-Apart from the tests above, other benchmarks can also focus on other performance metrics such as branch prediction or prefetching.
+This is a list of all the best cuisines in the world
+- Indian
+- Italian
+- French
 
-Other approaches are less artificial, measuring performance on real-world applications such as compile times and performance in the lastest (and most resource-demandign) video games.
-The latter metric revolves around how many average FPS (frames per second) a given CPU is able to crank out in a specific video game.
-[This article](https://www.gamersnexus.net/guides/3577-cpu-test-methodology-unveil-for-2020-compile-gaming-more) goes into more detail regarding the methodology of running CPU benchmarks on real-world applications.
+In order to make emphasize keywords, you can use **bold** words.
 
-Most benchmarks, unfortunately, are not open source, especially the more popular ones, such as [Geekbench 5](https://browser.geekbench.com/processor-benchmarks).
-Despite this shortcoming, benchmarks are widely used to compare the performance of various computer **hardware**, CPUs included.
+## Section the first
```

Comment:
```suggestion
## First Section
```

Reply:
![image](https://user-images.githubusercontent.com/19818392/197863010-d2b0fbd7-d47f-48dd-a798-26e394fafb57.png)

Then at least capitalise the word "first": "First".